.github/workflows/publish.yml

```diff
@@ -0,0 +1,45 @@
+name: Publish container
+on:
+  push:
+    tags:
+      - '*'
+jobs:
+  publish:
+    name: Build and push Spilo container
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+      packages: 'write'
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Docker meta
+      id: meta
+      uses: docker/metadata-action@v4
+      with:
+        # list of Docker images to use as base name for tags
+        images: |
+          ghcr.io/zalando/spilo-14
```

Comment:
Could it somehow be dynamic? I.e. extract the `PGVERSION` value from the Dockerfile and have something like `ghcr.io/zalando/spilo-$PGVERSION`.

Reply:
```sh
sed -n 's/^ARG PGVERSION=\([1-9][0-9]*\).*$/\1/p' Dockerfile
```